influxdb2_operational_monitoring/influxdb2_cardinality_now.yml

```diff
@@ -38,7 +44,7 @@ spec:
 
         buckets()
         	|> map(fn: (r) => {
-        		cards = influxdb.cardinality(start: -1d, bucket: r.name)
+        		cards = influxdb.cardinality(start: -1h, bucket: r.name)
```

Comment:
you should be able to set the start time to `-task.every`  which will be embedded into the flux script at runtime.